src/_json.js

```diff
@@ -42,6 +42,15 @@ function json_parse(_, val) {
     return new values.Bytes(val['@bytes'])
   } else if ('@query' in val) {
     return new values.Query(val['@query'])
+  } else if ('@access_provider' in val) {
+    var ref = val['@access_provider']
```

Comment:
This is not correct, `AccessProviders` are just a new reference type for drivers point of view

Reply:
@marrony Thanks. Should this be using `@ref` instead?

Reply:
you should do this as part of `@ref` parsing